router/handlers.go

```diff
@@ -1,20 +1,56 @@
 package router
 
 import (
+	"fmt"
 	"html/template"
 	"log"
 	"net/http"
+	"strings"
 )
 
 var (
 	AppTitle = "Budget Helper"
 )
 
 var (
-	tmplHome  = template.Must(template.ParseFiles("./templates/pages/home.html", "./templates/base.html"))
-	tmplLogin = template.Must(template.ParseFiles("./templates/pages/login.html", "./templates/base.html"))
+	tmplHome      = template.Must(template.ParseFiles("./templates/pages/home.html", "./templates/base.html"))
```

Comment:
Perhaps make a function that takes the template filename, "home.html", and injects it into this long line for you.